lib/req/request.ex

```diff
@@ -118,7 +118,7 @@ defmodule Req.Request do
     :url,
     headers: [],
     body: "",
-    adapter: {Req.Steps, :run_finch, []},
+    adapter: nil,
```

Comment:
wdyt about keeping the default that uses Finch here? If a bit of duplication is the main reason not to do it, I am not too concerned about that at the moment.

Reply:
I did this thinking that it is easier to know where to look for the defaults if they are set in one place, but now I see that the struct defaults match the defaults set in `build/3` (I missed this before), so I would say better to keep the default the way it was.